content/blog/2018/06/2018-06-19-new-design-digest.adoc

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "New design, UX and extensibility digest for login page et. al."
+tags:
+- core
+- developer
+author: scherler
+---
+
+== Target audience
```

Comment:
Maybe the header is not needed. Header without text does not look well in the blogpost render IIRC